src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/DocmosisTemplates.java

```diff
@@ -45,6 +45,7 @@ public enum DocmosisTemplates {
     HEARING_OTHER("CV-UNS-HNO-ENG-01196.docx", "hearing_other_claim_%s.pdf"),
     PIN_IN_THE_POST_LETTER("CV-CMC-LET-ENG-PIP0001.docx", "PIP_letter.pdf"),
     SET_ASIDE_JUDGMENT_IN_ERROR_LIP_DEFENDANT_LETTER("CV-CMC-LET-ENG-LIP-JO0001.docx", "set_aside_letter.pdf"),
+    JUDGMENT_BY_ADMISSION_PIN_LIP_DEFENDANT_LETTER("CV-CMC-LET-ENG-LIP-JO0005.docx", "judgment_by_admission_letter.pdf"),
```

Comment:
Is this is shown anywhere or it has no impact when document shown to user

Reply:
can we use this "PIN_IN_THE_POST_LETTER("CV-CMC-LET-ENG-PIP0001.docx", "PIP_letter.pdf")," or its template is different from the letter we are generating? It seems to be something generic.

Reply:
Have updated the name as per your suggestion.